warehouse/core/pom.xml

```diff
@@ -198,6 +198,16 @@
             <artifactId>easymock</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
junit-jupiter-engine has a dependency on junit-jupiter-api.  so you probably only need junit-jupiter-engine here.

Reply:
All the example docs I read had the explicit dependencies called out. I think that's better in the long run for readability.

Reply:
Not sure I agree with that.  That requires us to ensure we are using compatible versons.  It is simply more maintainance on our part.